src/components/FeatureViews/Registration/RegistrationActions.js

```diff
@@ -117,15 +121,15 @@ const RegistrationActions = () => {
 				onClose={closeRegisterMenu}
 				open={anchorEl !== null}
 			>
-				<MenuItem component={Link} to="/form/course_register/">
+				<MenuItem component={Link} to="/registration/form/class-registration">
 					<NewCourse className="icon innerIcon"/>
-					<ListItemText primary="COURSE"/>
+					<ListItemText primary="CLASS"/>
 				</MenuItem>
-				<MenuItem component={Link} to="/form/tutoring/">
+				<MenuItem component={Link} to="/registration/form/tutoring-registration/" disabled>
```

Comment:
Is this supposed to be permanently disabled?

Reply:
we're removing this flow - you won't be able to immediately register for a tutoring course now